api/models/sequelize/Passport.js

```diff
@@ -11,12 +10,20 @@ module.exports = {
       //More informations about supported models options here : http://docs.sequelizejs.com/en/latest/docs/models-definition/#configuration
       options: {
         hooks: {
-          beforeCreate: (values, options, fn) => {
-            hashPassword(app.config.passport.bcrypt, values, fn)
+          beforeCreate: (values, options) => {
+            if (values.password) {
```

Comment:
If you remove the callback you break retro compatibility with older versions. Would be better to keep the `hashPassword` method but something like `return hashPassword(app.config.passport.bcrypt, values, fn)` and then modify `hashPassword` to use callback if `fn` defined and use promise if it's not

Reply:
@jaumard fixed